src/components/KanaQuiz/KanaQuiz.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { KanaType } from '../../types/kana';
+import KanaQuizForm from './KanaQuizForm';
+import { QuizStats } from '../../types/kanaQuiz';
+
+type KanaQuizProps = {
+  kana: KanaType[];
+};
+
+export default function KanaQuiz({ kana }: KanaQuizProps) {
+  const [quizStats, setQuizStats] = useState({
```

Comment:
Does typescript currently know what type `quizStats` is? On a quick search, I've found this syntax for setting the type on useState(): `const [quizStats, setQuizStats] = useState<QuizStats>({...});`